.circleci/continue_config.yml

```diff
@@ -830,6 +839,13 @@ jobs:
     parameters:
       ms-edge:
         type: boolean
+        default: false
+      api:
+        type: boolean
+        default: false
+      case-details:
```

Comment:
This can be vague, wondering if this should be `api-case-endpoint` like the other parameters in other jobs?

Reply:
@tausifpatel is working on another CircleCI ticket, I'll make a note in the ticket 👍